chains/manager.go

```diff
@@ -1582,3 +1588,12 @@ func (m *manager) getOrMakeVMRegisterer(vmID ids.ID, chainAlias string) (metrics
 	)
 	return chainReg, err
 }
+
+// localParser intercepts invocations to ParseBlock and re-routes them to ParseLocalBlock
+type localParser struct {
+	*proposervm.VM
+}
+
+func (lp *localParser) ParseBlock(ctx context.Context, blockBytes []byte) (smcon.Block, error) {
+	return lp.ParseLocalBlock(ctx, blockBytes)
+}
```

Comment:
Thoughts on defining a `ParserFunc` type in the `block` package like:
```golang
type ParserFunc func(context.Context, []byte) (Block, error)

func (f ParserFunc) ParseBlock(ctx context.Context, blockBytes []byte) (Block, error) {
	return f(ctx, blockBytes)
}
```
Then we could initialize the `smbootstrap.Config` with:
```golang
Parser: block.ParserFunc(proposerVM.ParseLocalBlock)
```

If this seems unnecessary to export then we could move the definition here

Reply:
Done according to your suggestion